### PR TITLE
add attributes in model.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @vicancy

### CODE REVIEW

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace Microsoft.DocAsCode.Metadata.ManagedReference
 
     using Microsoft.DocAsCode.Common;
     using Microsoft.DocAsCode.DataContracts.ManagedReference;
-
+    using System.Collections.Immutable;
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs]: add one line

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs [hunk 2]
```diff
@@ -624,6 +638,89 @@ private ITripleSlashCommentParserContext GetTripleSlashCommentParserContext(Meta
             };
         }
 
+        private List<AttributeInfo> GetAttributeInfo(ImmutableArray<AttributeData> attributes)
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs]: check null?

**Comment by vicancy** [comment]
:+1:

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs (hunk 1 above)
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs]: fixed.

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs (hunk 2 above)
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs]: ImmutableArray<T> is structure.

**Pull request closed without merging**